_episodes/02-first-example.md

```diff
@@ -220,16 +222,19 @@ Options:
 
 ## Connect REANA client to remote REANA cluster
 
-The REANA client will interact with a remote REANA cluster.
+The REANA client will interact with a remote REANA cluster. Go to https://reana,cern.ch to activate your REANA token.
```

Comment:
typo in URL

Reply:
"obtain a REANA token"

Reply:
I actually think this is descried better a few lines down: "The authentication uses a token that one can get..." etc. 
I tink the order should be:

The REANA client knows to which REANA cluster it connects by means of two environment variables:
code (removing the REANA_ACCESS_TOKEN line)
In order to authenticate to REANA, you need to generate a token.
Exercise: Obtain a token.
In order to set the token, run 
code block: export REANA_ACCESS_TOKEN=xxxxxx

The REANA client connection to remote REANA cluster can be verified via ``ping`` command: